input/fsh/Profiles clinical datapoints/TumorFactors.fsh

```diff
@@ -0,0 +1,394 @@
+// HISTOTYPE
+Profile: Histotype
+Parent: Observation 
+Id: histo-type
+Title: "Histological type of the tumor"
+Description: "Represents the histological type of the tumor"
+* insert PublicationProfileRuleset
+* code = SCT#396984004 "Histologic feature of neoplasm (observable entity)"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from HistologicalTypeVS (required)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: HistotypePatient147
+InstanceOf: Histotype
+Description: "Example of the histological type of the breast cancer tumor"
+* status = ObservationStatusCS#final
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = SCT#722524005
+
+Mapping: HistotypeToICHOM
+Source:	Histotype
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: histotypemapping
+Title: "Histotype to ICHOM set"
+Description: "Mapping of the histologival type of tumor to the ICHOM breast cancer PCOM set" 	
+* valueCodeableConcept -> "Histological type"
+
+// GENETIC MUTATION
+Profile: GermlineMutation
+Parent: Observation 
+Id: genetic-mutation
+Title: "Germline mutation"
+Description: "Represents if the patient carries a germline mutation predisposing breast cancer"
+* insert PublicationProfileRuleset
+* code = SCT#55446002 "Genetic mutation"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from GermlineMutationVS (required)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: GermlineMutationPatient147
+InstanceOf: GermlineMutation
+Description: "Example of whether a patient carries a germline mutation predisposing breast cancer"
+* status = ObservationStatusCS#final
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = SCT#445180002 
+
+Mapping: GermlineMutationToICHOM
+Source:	GermlineMutation
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: GermlineMutationMapping
+Title: "Germline mutation to ICHOM set"
+Description: "Mapping of the germline mutation to the ICHOM breast cancer PCOM set" 	
+* valueCodeableConcept -> "Germline mutation"
+
+// TUMOR GRADING
```

Comment:
I see that I haven't been very consistent with adding effectiveDateTime as MS. 

For the profile histotype I have an example of the date because the date of histological diagnosis is explicitly in the ICHOM set (as an element). For the other profiles the date is not explicitly in the ICHOM set. However, timing is an important factor in the set, because the data is repeatedly collected during follow-up of the patient. So I figured the timing could be relevant to understand the timing of these observations, that's why I wanted to profile them as MS, but forgot to add them to all the profiles. For the instances I don't have example dates, but I could reuse the date of the histological diagnosis. 

In short: Do we want effectiveDateTime as MS in all the observations? And if yes, do we want example dates as well?

What do you think, @vadi2